<launch>

    <rosparam ns="/agv/mapfile" command="load" file="$(find webclient)/robot_config/defaultMap.yaml"/>
    <node name="map_server" pkg="map_server" type="map_server" 
    launch-prefix="/bin/bash -c '$* `rosparam get /agv/mapfile`' --">
     <param name="frame_id" type="string" value="map"/>
    </node>

</launch>
